### PR TITLE
Add Winget Releaser workflow

### PR DESCRIPTION
**Pull request by sitiom**
This action automatically generates manifests for Winget Community Repository ([microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs)) and submits them.

Flow Launcher is already in @vedantmgoyal2009's automation, as stated in https://github.com/Flow-Launcher/Flow.Launcher/pull/1061#issue-1158217797, but the author now urges for GitHub packages to use the action instead. See https://github.com/vedantmgoyal2009/vedantmgoyal2009/issues/931

**Before merging this:**
1. Add a PAT with `public_repo` scope as a repository secret named `WINGET_TOKEN` (or rename the secret name in the workflow).

example

2. Fork https://github.com/microsoft/winget-pkgs under @Flow-Launcher. The action will use that fork for making a branch and creating a PR with the upstream [winget-pkgs](https://github.com/microsoft/winget-pkgs) repository on every release.
3. Install [Pull](https://github.com/apps/pull) on the winget-pkgs fork to ensure that it is constantly updated.

The documentation and source code of the action is available [here](https://github.com/vedantmgoyal2009/winget-releaser/).

If you want to see an example of a PR created using this action, see [microsoft/winget-pkgs/pulls (Pull request has been automatically created using 🛫 WinGet Releaser)](https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+sort%3Aupdated-desc+Pull+request+has+been+automatically+created+using+%F0%9F%9B%AB+WinGet+Releaser).

### CODE REVIEW

**Comment by sitiom** [comment]
Bump

**Comment by jjw24** [comment]
Hey @sitiom, thanks for the PR, was going to take a look after we released 1.10.0, so will look to set it up soon.

**Comment by VictoriousRaptor** [comment]
@jjw24 is this pr ready?

**Comment by jjw24** [comment]
No, I believe there are other settings need to be done.

**Comment by sitiom** [comment]
This PR is technically ready; you just need to do the prerequisites.

**Comment by jjw24** [comment]
Let me have a look.

**Comment by jjw24** [comment]
Prerequisites all done.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -15,3 +15,7 @@ updates:
       - "jjw24"
       - "taooceros"
       - "JohnTheGr8"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (jjw24) [inline .github/dependabot.yml:21]:
```suggestion
      interval: "daily"
```

Comment (jjw24) [reply .github/dependabot.yml:21]: Just changing this to daily

**Review by jjw24 — APPROVED** [review]

**Pull request merged**